packages/components/src/higher-order/with-focus-return/index.js

```diff
@@ -79,7 +79,7 @@ function withFocusReturn( options ) {
 				// the default behavior otherwise occurring here. This allows
 				// for support that the `onFocusReturn` decides to allow the
 				// default behavior to occur under some conditions.
-				if ( onFocusReturn() === false ) {
+				if ( onFocusReturn( this.props.childProps ) === false ) {
```

Comment:
I can't find any references to this new prop. Does it fix some bugs?

Reply:
I think this change was not needed anymore It was part of an experiment and I missed the removal.